client/src/memberSearch/profilePage/ProfilePage.js

```diff
@@ -206,7 +208,7 @@ function ProfilePage(props) {
                     </div>
 
                 </div>
-                {sendingMessage && <SendMessage otherUser={username} username={"messageMember1"}/>}
+                {sendingMessage && <SendMessage receiverName={username} receiverId={uid}/>}
```

Comment:
Well renamed.

Reply:
Thanks. It was necessary to make sure props made sense